cpu/native/socket_zep/socket_zep.c

```diff
@@ -36,6 +36,10 @@
 #define ENABLE_DEBUG 0
 #include "debug.h"
 
+#ifndef SOCKET_ZEP_AUTO_ACK
+#define SOCKET_ZEP_AUTO_ACK 1
```

Comment:
socket_zep is a pure software implementation, if the submac now provides an auto-ACK mechanism just drop it from `socket_zep` entirely - we don't need to maintain two code paths and the main goal of `socket_zep` is to test the common infrastructure, so we want to keep it as minimal as possible.

Reply:
For easier review and testing I did not delete it yet to make it easy to switch